modules/apps/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/service/impl/MBMessageLocalServiceImpl.java

```diff
@@ -2127,7 +2127,9 @@ private MBSubscriptionSender _getSubscriptionSender(
 			String messageBody, String messageSubject,
 			String messageSubjectPrefix, String inReplyTo, String fromName,
 			String fromAddress, String replyToAddress, String emailAddress,
-			String fullName, LocalizedValuesMap subjectLocalizedValuesMap,
+			String fullName, String messageParent, String messageSiblings,
```

Comment:
"messageParent" and "messageSibling" need to be renamed. The former implies that it contains an MBMessage, and the latter that it is a collection. Rename these to something that unambiguously says what they are (e.g. "messageParentMessageContent" and "messageSiblingMessagesContent".)